components/common/srtarterpacks/RecordCard.tsx

```diff
@@ -0,0 +1,166 @@
+import { useQuery } from '@tanstack/react-query'
+import Link from 'next/link'
+import { Loading, LoadingFailed } from '@/common/Loader'
+import client from '@/lib/client'
+import { buildBlueSkyAppUrl, parseAtUri } from '@/lib/util'
+import { AppBskyGraphDefs } from '@atproto/api'
+
+export const StarterPackRecordCard = ({ uri }: { uri: string }) => {
+  const { error, data, isFetching } = useQuery({
+    retry: false,
+    queryKey: ['starterpack', uri],
+    queryFn: async () => {
+      const { data } = await client.api.app.bsky.graph.getStarterPack(
+        {
+          starterPack: uri,
+        },
+        { headers: client.proxyHeaders() },
+      )
+      return data
+    },
+  })
+
+  if (isFetching) {
+    return <Loading />
+  }
+
+  if (error) {
+    return <LoadingFailed error={error} />
+  }
+
+  if (!data) {
+    return <LoadingFailed error="Feed generator data not found!" />
+  }
+
+  const {
+    starterPack: {
+      creator,
+      list,
+      listItemsSample,
+      feeds,
+      joinedWeekCount,
+      joinedAllTimeCount,
+      labels,
+      indexedAt,
+      record,
+    },
+  } = data
+  const displayName = record['name'] || 'Untitled'
+
+  const meta: string[] = [
+    `${joinedWeekCount} Joined last week`,
+    `${joinedAllTimeCount} Joined all time`,
+    `${feeds?.length || 'no'} Feed included`,
+  ]
+
+  if (indexedAt) {
+    meta.push(new Date(indexedAt as string).toLocaleString())
+  }
+
+  return (
+    <div className="bg-white dark:bg-slate-800">
+      <div className="flex w-full space-x-4">
+        <div className="flex-shrink-0">
+          <img
+            className="h-6 w-6 rounded-full"
+            // Probably should use a different default avatar for all feed generator app-wide
+            src={creator.avatar || '/img/default-avatar.jpg'}
+            alt={`Avatar of the creator of starter pack ${displayName}`}
+          />
+        </div>
+        <div className="min-w-0 flex-1">
+          <p className="text-sm font-medium text-gray-900 dark:text-gray-200">
+            <>
+              <Link
+                href={`/repositories/${uri?.replace('at://', '')}`}
+                className="hover:underline"
+              >
+                <span className="font-bold">{displayName}</span>
+              </Link>
+              <span className="ml-1">by</span>
+              <Link href={`/repositories/${creator.did}`}>
+                <span className="ml-1 text-gray-500 dark:text-gra-50">
+                  @{creator.handle}
+                </span>
+              </Link>
+            </>{' '}
+            &nbsp;&middot;&nbsp;
+            <a
+              href={buildBlueSkyAppUrl({ did: creator.did })}
+              target="_blank"
+              rel="noreferrer"
+            >
+              Peek
+            </a>
+          </p>
+        </div>
+      </div>
+      <div className="pb-2 pl-10">
+        {!!feeds?.length && (
```

Comment:
Ah totally forgot about this!  Not a blocking issue, but another thing we may want to show is the og card preview of the starter pack.  It would be useful for mods to see if it contains an inappropriate avatar image, or something along those lines.  They live at `https://ogcard.cdn.bsky.app/start/{did}/{rkey}`.  E.g. https://ogcard.cdn.bsky.app/start/did:plc:ragtjsm2j2vknwkz3zp4oxrd/3kvaqmutnga2z